packages/nuxt-module/src/module.ts

```diff
@@ -0,0 +1,110 @@
+import process from 'node:process'
+
```

Comment:
this file is very similar to the vite plugin implementation. it was copied and adjusted to use nitro hooks.